plugins/vpp/l3plugin/vppcalls/vpp2001/dhcpproxy_dump.go

```diff
@@ -37,12 +37,12 @@ func (h *DHCPProxyHandler) DumpDHCPProxy() ([]*vppcalls.DHCPProxyDetails, error)
 		}
 		proxy := &l3.DHCPProxy{
 			RxVrfId:         dhcpProxyDetails.RxVrfID,
-			SourceIpAddress: net.IP(dhcpProxyDetails.DHCPSrcAddress[:4]).To4().String(),
+			SourceIpAddress: addressToString(dhcpProxyDetails.DHCPSrcAddress),
```

Comment:
oh no. With this `addressToString` function loops for `IsIp6: false` and `IsIp6: ture` become completely identical.

Reply:
Nice catch! I guess we could define a function for processing dump replies and use that in both cases.